wrappers/python/tests/layer/test_embedding.py

```diff
@@ -81,3 +92,65 @@ def test_embedding(dtype: np.dtype):
     abs_error = np.linalg.norm(np_vocab_torch - np_vocab)
     rel_error = abs_error / np.linalg.norm(np_vocab_torch)
     assert rel_error < 1e-6
```

Comment:
Irrelevant changes above this line.

Reply:
just ruff
If you belive I have some problem with formatter settings - can you give config please?